hbase-commit-table/src/main/java/com/yahoo/omid/committable/hbase/HBaseCommitTable.java

```diff
@@ -86,9 +87,11 @@ protected HBaseCommitTable(Configuration hbaseConfig, String tableName, KeyGener
 
     public class HBaseWriter implements Writer {
 
+        private static final long INITIAL_LWM_VALUE = -1L;
         final HTable table;
         // Our own buffer for operations
         final List<Put> writeBuffer = new LinkedList<>();
+        final AtomicLong lowWatermarkToStore = new AtomicLong(INITIAL_LWM_VALUE);
```

Comment:
I've added because two concurrent threads could potentially access that variable in HA scenarios, the thread in the RequestProcessor and the thread in the HA lease manager initializing the tso state through the reset() method